### PR TITLE
Propay

### PR DESCRIPTION
**Pull request by kingfishbrain**
Methods aren't tested yet

### CODE REVIEW

File: ausleih-service/src/main/java/hhu/ausleihservice/propay/ProPayInterface.java [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+		LinkedMultiValueMap map = new LinkedMultiValueMap();
+		map.add("sourceAccount", sourceAccount);
+		map.add("targetAccount", targetAccount);
+		map.add("amount", amount);
+
+		try {
+			String response = WebClient
+					.create()
+					.post()
+					.uri(builder -> builder.scheme("httpp")
```
Comment (siker102) [inline ausleih-service/src/main/java/hhu/ausleihservice/propay/ProPayInterface.java]: httpp -> http

File: ausleih-service/src/main/java/hhu/ausleihservice/propay/ProPayInterface.java [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+							.pathSegment("account", sourceAccount, "transfer", targetAccount)
+							.build())
+					.body(BodyInserters.fromMultipartData(map))
+					.accept(MediaType.APPLICATION_JSON_UTF8)
+					.retrieve()
+					.bodyToMono(String.class)
+					.block();
+		} catch (Exception e) {
+			e.printStackTrace();
+		}
```
Comment (siker102) [inline ausleih-service/src/main/java/hhu/ausleihservice/propay/ProPayInterface.java:73]: return statement missing

**Comment by zn5** [comment]
All Methods have tests and work properly now

**Comment by burakmartin** [comment]
+1

**Pull request merged**